### PR TITLE
Add column row

### PR DESCRIPTION
**Pull request by OnderUlasKilic**
Add colum row

### CODE REVIEW

File: lib/101/column_row_learn.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                  Expanded(child: Text("data")),
+                ],
+              ))
+        ],
+      ),
+    );
+  }
+}
+
+class ProhectContainerSizes {
```
Comment (BBarisKilic) [inline lib/101/column_row_learn.dart]: Fix the typo.

```suggestion
class ProjectContainerSizes {
```

File: lib/101/column_row_learn.dart [hunk 2]
```diff
@@ -0,0 +1,52 @@
+// ignore_for_file: unused_import
```
Comment (BBarisKilic) [inline lib/101/column_row_learn.dart]: Remove unnecessary line.

File: lib/core/random_image.dart [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// ignore_for_file: unused_import
```
Comment (BBarisKilic) [inline lib/core/random_image.dart]: Remove unnecessary line.

File: lib/core/random_image.dart [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+import 'package:flutter/material.dart';
+
+class RandomImage extends StatelessWidget {
+  const RandomImage({super.key, this.height = 100});
+  final imageUrl = "https://picsum.photos/200/300";
```
Comment (BBarisKilic) [inline lib/core/random_image.dart]: Prefer static const here.

```suggestion
  static const imageUrl = "https://picsum.photos/200/300";
```

File: lib/core/random_image.dart [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// ignore_for_file: unused_import
+
+import 'package:flutter/material.dart';
+
+class RandomImage extends StatelessWidget {
+  const RandomImage({super.key, this.height = 100});
```
Comment (BBarisKilic) [inline lib/core/random_image.dart:4]: And in general, give a space between constructor and class variables to improve readability.

```suggestion
  const RandomImage({super.key, this.height = 100});
  
  final imageUrl = "https://picsum.photos/200/300";
```

File: lib/101/column_row_learn.dart [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+          Expanded(
+              flex: 4,
+              child: Row(
+                children: [
+                  Expanded(child: Container(color: Colors.red)),
+                  Expanded(child: Container(color: Colors.green)),
+                  Expanded(child: Container(color: Colors.yellow)),
+                  Expanded(child: Container(color: Colors.pink)),
+                ],
+              )),
```
Comment (BBarisKilic) [inline lib/101/column_row_learn.dart]: Add comma where applicable to improve readability.

```suggestion
              ),
             ),
```

File: lib/101/custom_widget_learn.dart [hunk 7]
```diff
@@ -30,6 +30,21 @@ class CustomWidgetLearn extends StatelessWidget {
             title: title,
             onPressed: () {},
           ),
+          const SizedBox(height: 20),
+          ElevatedButton(
+            style: ElevatedButton.styleFrom(
+                backgroundColor: _ColorsUtility.red,
+                shape: const StadiumBorder()),
```
Comment (BBarisKilic) [inline lib/101/custom_widget_learn.dart]: Add comma where applicable to improve readability.

```suggestion
                shape: const StadiumBorder(),
                ),
```

File: lib/101/column_row_learn.dart [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+          const Spacer(flex: 1),
+          const SizedBox(
+              height: ProhectContainerSizes.cardHeight,
+              child: Column(
+                children: [
+                  Expanded(child: Text("data")),
+                  Expanded(child: Text("data")),
+                  Expanded(child: Text("data")),
+                  Expanded(child: Text("data")),
```
Comment (BBarisKilic) [inline lib/101/column_row_learn.dart]: Add comma where applicable to improve readability.

```suggestion
                  Expanded(child: Text("data"),
                  ),
                  Expanded(child: Text("data"),
                  ),
                  Expanded(child: Text("data"),
                  ),
                  Expanded(child: Text("data"),
                  ),
```

File: lib/101/column_row_learn.dart [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+      body: Column(
+        children: [
+          Expanded(
+              flex: 4,
+              child: Row(
+                children: [
+                  Expanded(child: Container(color: Colors.red)),
+                  Expanded(child: Container(color: Colors.green)),
+                  Expanded(child: Container(color: Colors.yellow)),
+                  Expanded(child: Container(color: Colors.pink)),
```
Comment (BBarisKilic) [inline lib/101/column_row_learn.dart]: Add comma where applicable to improve readability.

```suggestion
                  Expanded(child: Container(color: Colors.red,),),
                  Expanded(child: Container(color: Colors.green,),),
                  Expanded(child: Container(color: Colors.yellow,),),
                  Expanded(child: Container(color: Colors.pink,),),
```

**Review by BBarisKilic — CHANGES_REQUESTED** [review]
The new additions look fine. I just suggested some small readability improvement ways.

File: lib/101/column_row_learn.dart (hunk 1 above)
Comment (OnderUlasKilic) [reply lib/101/column_row_learn.dart]: Done.

File: lib/101/column_row_learn.dart (hunk 2 above)
Comment (OnderUlasKilic) [reply lib/101/column_row_learn.dart]: Done.

File: lib/core/random_image.dart (hunk 3 above)
Comment (OnderUlasKilic) [reply lib/core/random_image.dart]: Done.

File: lib/core/random_image.dart (hunk 4 above)
Comment (OnderUlasKilic) [reply lib/core/random_image.dart]: Done.

File: lib/core/random_image.dart (hunk 5 above)
Comment (OnderUlasKilic) [reply lib/core/random_image.dart:4]: Done.

**Review by BBarisKilic — APPROVED** [review]

**Pull request merged**